kit/app/views/current/r2/603-5829-4.html

```diff
@@ -0,0 +1,151 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  Check an Early Years qualification – {{ serviceName }} – GOV.UK Prototype Kit
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: "javascript:window.history.back()"
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+
+      <span class="govuk-caption-xl gem-c-title__context">
+       Early Years Qualifications List
+      </span>
+
+      <h1 class="govuk-heading-xl">
+        T Level Technical Qualification in Education and Childcare (Specialism - Early Years Educator)
+      </h1>
+
+      {{ govukSummaryList({
+        rows: [
+
+        {
+            key: {
+              text: "Awarding organisation"
+            },
+            value: {
+              text: "iFATE"
+            },
+            actions: {
+
+            }
+          }
+          ,
+          {
+            key: {
+              text: "Level"
+            },
+            value: {
+              text: "Level 3"
+            },
+            actions: {
+
+            }
+          }
+          ,
+          {
+            key: {
+              text: "Date added to the early years qualifications list"
+            },
+            value: {
+              text: "1 September 2020"
+            },
+            actions: {
+
+            }
+          }
+          ,
+          {
+            key: {
+              text: "Date started"
+            },
+            value: {
+              text: "6 September 2021"
+            },
+            actions: {
+
+            }
+          }
+          ,
+          {
+            key: {
+              text: "Date of check"
+            },
+            value: {
+              text: "21 February 2024"
+            },
+            actions: {
+
+            }
+          }
+        ]
+      }) }}
+
+      <p>
+        This qualification is approved by the Department for Education as full and relevant, subject to the following notes:
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>The course must be assessed within the early years foundation stage (EYFS) in an early years setting in England. </li>
+        </ul>
+      </p>
+
+      <p>
+        This is a level 3 qualification which was started after 1 September 2014. Someone who holds this qualification can be included in staff:child ratios at level 3, as long as they meet all other requirements as set out in the <a href="https://www.gov.uk/government/publications/early-years-foundation-stage-framework--2">Early years foundation stage statutory framework (EYFS)</a> and the <a href="https://www.gov.uk/government/publications/early-years-qualification-requirements-and-standards">Early years qualification requirements and standards</a> document.
```

Comment:
Does the word 'Early' after the link need to be lowercase?

Reply:
```suggestion
        This is a level 3 qualification which was started after 1 September 2014. Someone who holds this qualification can be included in staff:child ratios at level 3, as long as they meet all other requirements as set out in the <a href="https://www.gov.uk/government/publications/early-years-foundation-stage-framework--2">early years foundation stage statutory framework (EYFS)</a> and the <a href="https://www.gov.uk/government/publications/early-years-qualification-requirements-and-standards">early years qualification requirements and standards</a> document.
```